app.js

```diff
@@ -0,0 +1,87 @@
+var express = require('express')
+var http = require('http')
+var path = require('path')
+var passport = require('passport')
+var LocalStrategy = require('passport-local').Strategy
+
+//==================================================================
+// Define the strategy to be used by PassportJS
+passport.use(new LocalStrategy(
+  function(username, password, done) {
+    // @todo Tie this authentication to the members database in CouchDB
+    if (username === "admin" && password === "admin") // stupid example
+      return done(null, {name: "admin"})
+
+    return done(null, false, { message: 'Incorrect username.' })
+  }
+))
```

Comment:
Passport handles sessions for us so we can do things to check the user's session like [here](https://github.com/open-learning-exchange/BeLL-Apps/pull/37/files#diff-0364f57fbff2fabbe941ed20c328ef1aR70). This function is what is run when authenticating a user using the `/login` route [here](https://github.com/open-learning-exchange/BeLL-Apps/pull/37/files#diff-0364f57fbff2fabbe941ed20c328ef1aR74). Currently this function just checks to see if the username and password provided is "admin" and "admin" but we could easily tie this to document lookups in CouchDB's `members` database.